src/Tutorials/Basics/HDA_flowsheet_solution_testing.ipynb

```diff
@@ -938,7 +938,7 @@
    "outputs": [],
    "source": [
     "def function(unit):\n",
-    "        unit.initialize(outlvl=idaeslog.INFO)"
+    "        unit.initialize(outlvl=idaeslog.DEBUG)"
```

Comment:
This should be left at the INFO level - we do not want debugging output in an example notebook.